Multisig/Logic/Models/Safe.swift

```diff
@@ -47,6 +49,17 @@ extension Safe: Identifiable {
         return (try? context.fetch(Safe.fetchRequest().all())) ?? []
     }
 
+    static func updateCachedNames() {
```

Comment:
It was because you mentioned that we can cache them, and because it is easy to do so. Probably working directly with database will not be an issue at all. But I planned to refactor it to AddressBook service anyways once we have multiple signing keys with names, safe names and etc.